gcache/src/gcache_rb_store.cpp

```diff
@@ -411,6 +424,8 @@ namespace gcache
     void
     RingBuffer::estimate_space(bool zero_out)
     {
+        size_rnd_ = 0;
```

Comment:
Why?

Reply:
It is used in seqno_reset() and recover() methods where we have no released, but not discarded buffers.

Reply:
OK, I have added assert here, and put zeroing of size_rnd_ and detailed explanation why we need to do it this way in RingBuffer::seqno_reset() caller.